pkg/operators/v1alpha1/clusterserviceversion_types.go

```diff
@@ -148,8 +148,12 @@ type APIServiceDescription struct {
 // APIResourceReference is a Kubernetes resource type used by a custom resource
 // +k8s:openapi-gen=true
 type APIResourceReference struct {
+	// Name of the CustomResourceDefinition for the referent, or an empty string when the referent
+	// is not a custom resource
 	Name    string `json:"name"`
+	// Kind of the referent
 	Kind    string `json:"kind"`
+	// API version of the referent
```

Comment:
Full stops are missing in these comments too but at least it's consistent with the comment for the containing type...